front/pages/api/v1/w/[wId]/triggers/hooks/[webhookSourceId]/index.ts

```diff
@@ -143,6 +143,20 @@ async function handler(
     });
   }
 
+  // Validate webhook url secret
+  if (
+    typeof query.secret !== "string" ||
+    query.secret !== webhookSource.urlSecret
```

Comment:
we store the hardcoded version right ? should we store a hash of the secret and check against the hash ? 

prevents us from any db leak / extraction

Reply:
We still need to display it back to the user, so the implem might be a bit harder (we need to store the actual secret somewhere then revoke it)
But I agree storing the real secret forever is not ideal in term of security

Reply:
@adrsimon @fabiencelier I don't see the point since we generate this secet. So saving a hash or a generated secret won't differ, because in any cases we display it (the hash or the generated secret) to the admin in the url...

Reply:
What about we store the secret and the hash in diffrent columns.
We store the hash permanently and use it for validating the input secret
We only store and display the secret for 10 minutes, then we remove it / hide all chars but the last 4 

This depends on whether we want to display the URL with secret forever or just for 10 minutes
I'm totally ok to delay this discussion to later and merge the PR as is